src/main/java/com/google/api/codegen/transformer/ApiMethodTransformer.java

```diff
@@ -350,7 +434,7 @@ private ApiMethodDocView generateOptionalArrayMethodDoc(MethodTransformerContext
 
     docBuilder.mainDocLines(context.getNamer().getDocLines(context.getMethod()));
     List<ParamDocView> paramDocs =
-        getMethodParamDocs(context, context.getMethodConfig().getRequiredFields());
+        getMethodParamDocs(context, context.getMethodConfig().getRequiredFields(), null);
```

Comment:
I'd also prefer this to either pass an empty list or use an overload instead of using null.